site/content/docs/main/deploy-options.md

```diff
@@ -176,9 +189,9 @@ Next, pass `--envoy-service-http-port=80 --envoy-service-https-port=443` to the
 This is best paired with a DaemonSet (perhaps paired with Node affinity) to ensure that a single instance of Contour runs on each Node.
 See the [AWS NLB tutorial][10] as an example.
 
-### Upgrading Contour/Envoy
+## Upgrading Contour/Envoy
 
-At times it's needed to upgrade Contour, the version of Envoy, or both.
+At times, it's needed to upgrade Contour, the version of Envoy, or both.
 The included `shutdown-manager` can assist with watching Envoy for open connections while draining and give signal back to Kubernetes as to when it's fine to delete Envoy pods during this process.
 
 See the [redeploy envoy][11] docs for more information.
```

Comment:
similar to previous comment, this section covers some conceptual stuff around upgrading, but doesn't actually link to the "how to upgrade" page. We should figure out how to cover *both* conceptual stuff and nuts and bolts of upgrading. Again, probably a separate issue.